sysfs/gpio.go

```diff
@@ -431,7 +431,7 @@ func (d *driverGPIO) String() string {
 }
 
 func (d *driverGPIO) Prerequisites() []string {
-	return []string{"ioctl-gpio"}
+	return nil
```

Comment:
If there's really some dependency on `ioctl-gpio` we could move it to the `After()` below...!?